core/src/main/java/org/bitcoinj/wallet/Wallet.java

```diff
@@ -1668,12 +1668,25 @@ public Context getContext() {
      *
      * @param file the wallet file to read
      * @param walletExtensions extensions possibly added to the wallet.
+     * @return The Wallet
+     * @throws UnreadableWalletException
      */
     public static Wallet loadFromFile(File file, @Nullable WalletExtension... walletExtensions) throws UnreadableWalletException {
-        try {
-            try (FileInputStream stream = new FileInputStream(file)) {
-                return loadFromFileStream(stream, walletExtensions);
-            }
+        return loadFromFile(file, false, false, walletExtensions);
+    }
+
+    /**
```

Comment:
There is an incomplete JavaDoc here.

Reply:
Fixed.

Reply:
It's still missing the overall description and the description of the `ignoreMandatoryExtensions` parameter.

Reply:
That should be resolved. For some reason that latest force push is not showing here. I'm going to mark resolved.

Reply:
Ignore my previous confusion, I'll fix and force push...